discovery-cloud-secure.md

```diff
@@ -0,0 +1,30 @@
+
+# Discover using the cloud, but otherwise still secure
+## QR + plex + LE
+  * (box will temporarily set up the bridge and update DNS during LE reg/renew)
+  * con: you need to trust not only the Box and the client, but also the bridge/DNS-server
+  * pro: no need for Cordova; just use a standard QR-code reader and mobile browser
```

Comment:
EDIT: No separate QR-code reader needed if we include https://webqr.com/ into the app.